tests/kafkatest/services/kafka/kafka.py

```diff
@@ -508,4 +508,20 @@ def get_offset_shell(self, topic, partitions, max_wait_ms, offsets, time):
         for line in node.account.ssh_capture(cmd):
             output += line
         self.logger.debug(output)
+        return output
+
+    def get_simple_consumer_shell(self, topic, partition=0):
```

Comment:
I too initially thought the same and was trying to add it as a service, but it does not have characteristics of a long running service. The tool simply prints offsets of concerned partitions and exits.